charts/argo-cd/templates/argocd-server/ingress.yaml

```diff
@@ -2,7 +2,11 @@
 {{- $serviceName := include "argo-cd.server.fullname" . -}}
 {{- $servicePort := .Values.server.service.servicePortHttp -}}
 {{- $paths := .Values.server.ingress.paths -}}
+{{- if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
+apiVersion: networking.k8s.io/v1beta1
+{{- else -}}
 apiVersion: extensions/v1beta1
+{{- end }}
```

Comment:
A more explicit way of evaluating this is using `.Capabilities.APIVersions.Has`

```
{{- if .Capabilities.APIVersions.Has "networking.k8s.io/v1beta1" }}
apiVersion: networking.k8s.io/v1beta1
{{ else }}
apiVersion: extensions/v1beta1
{{ end -}}
```

Reply:
Awesome, didn't know about this. That's definitely clearer.